### DISCUSSION TITLE
ENOENT err: no such file or directory, mkdir. This is related to npm not being able to find a file.

### DISCUSSION THREAD

**Discussion by kushagra6776** [nodejs/help#4753] [Q&A]
I am getting this error for long, I have already reinstalled nodejs and reactjs more than 10 times, still nothing changed. My npm commands are also not working neither I am able to create a new app. I have tried "npm cache clean", "npm config edit", "npm cache clean --force", "npm audit" etc, but none responded. Please suggest me some way to resolve this issue!!

**Comment by ishan-18** [comment]
Verify PATH environment variable:
Ensure that the Node.js and npm directories are added to your PATH environment variable. If they are not, you may need to add them manually. The process for setting the PATH variable varies based on your operating system.

Firewall or antivirus:
Check if any firewall or antivirus software is blocking Node.js or npm from accessing the internet. Temporarily disable them and see if it resolves the issue.
